debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -52,6 +52,10 @@ public Class<? extends Task> taskClass() {
 
     @Override
     public void start(Map<String, String> props) {
+        if (props.getOrDefault(PostgresConnectorConfig.SHOULD_FLUSH_LSN_IN_SOURCE_DB.name(), "true").equalsIgnoreCase("false")) {
```

Comment:
Can you please do this check in `POstgresConnectorConfig` in a dedicated validation method attached directly to `SHOULD_FLUSH_LSN_IN_SOURCE_DB` that would just emit warning and not fail?